tests/install/openqa_webui.pm

```diff
@@ -17,35 +17,41 @@ sub install_from_repos {
     $add_repo = "zypper -n ar -p 95 -f obs://devel:openQA/$repo openQA";
     assert_script_run($_) foreach (split /\n/, $add_repo);
     assert_script_run('retry -s 30 -- sh -c "zypper -n --gpg-auto-import-keys ref && zypper --no-cd -n in openQA-local-db"', 600);
-    my $configure = <<'EOF';
-/usr/share/openqa/script/configure-web-proxy
-sed -i -e 's/#.*method.*OpenID.*$/&\nmethod = Fake/' /etc/openqa/openqa.ini
-systemctl restart apache2
-systemctl enable --now openqa-webui
-systemctl status --no-pager openqa-webui
-systemctl enable --now openqa-scheduler
-systemctl status --no-pager openqa-scheduler
-EOF
-    assert_script_run($_) foreach (split /\n/, $configure);
+    if (check_var('OPENQA_WEB_PROXY', 'nginx')) {
+        assert_script_run('retry -s 30 -- sh -c "zypper --no-cd -n in nginx"', 600);
```

Comment:
Instead of this I suggest to use a sub-package that does not install apache but does install nginx. Then we don't have two subsequent zypper install calls but just one.

Reply:
A subpackage that would just recommend nginx and do nothing else would be a bit much overhead.
I will update the PR with a simpler solution.

Reply:
Actually we can change to use "openQA-single-instance" in the case of apache2 and have a corresponding "openQA-single-instance-nginx" in the case of nginx. See https://github.com/os-autoinst/openQA/blob/master/dist/rpm/openQA.spec

Reply:
Yes, but here we currently only install the webui. The worker is set up in a separate step.
If we pull in the worker via the single instance rpm, the actual worker test module might have redundant steps then.

This doesn't mean that it doesn't make sense to have a single-instance nginx rpm. I just would not use it here.
Also it would install both nginx and apache unless installed with `--no-recommends` as the openQA rpm recommends apache.